TodoApp/TodoApp.Services.Tests/Updaters/ItemUpdaterTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using MongoDB.Driver;
+using NSubstitute;
+using NUnit.Framework;
+using TodoApp.Contract.Models;
+using TodoApp.Contract.Services.Generators;
+using TodoApp.Contract.Services.Updaters;
+using TodoApp.Contract.Tests.Utilities;
+using TodoApp.Services.Updaters;
+
+namespace TodoApp.Services.Tests.Updaters
+{
+    internal class ItemUpdaterTests : TestBase
```

Comment:
Once you change the ItemUpdater these tests needs to be refactored. Look at ItemCreator tests for inspiration.